src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -472,7 +498,7 @@ public WorkspacePanel(final MCreator mcreator) {
 		leftPan.add(sort);
 
 		se.add("West", leftPan);
-
+		

```

Comment:
Why this indent?

Reply:
Fixed the whitespace, there is a space here by default in main code, i just removed it accidently but it's fine now